samples/zigbee/light_switch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+add_definitions(-DZB_ED_ROLE)
+
+include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Should be `find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})`
And depending on whether this comes in before / after: #2232 
then the first might be obsolete.

Reply:
fixed by rebase